CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.10.0 - 2018-11-28
+### Added
+- CosmosDB Connections: Leverage CosmosDB Extension (https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-cosmosdb) to effortlessly connect databases to web apps
+- Update default deploy settings option
+- "Connect to Log Stream" node within "Logs"
+- Deployments node for Web Apps that are connected to a GitHub or LocalGit repository
```

Comment:
move deployments to the top. I think it's the most exciting

Also add more detail to this. Something like 
```
- Deployments node for Web Apps that are connected to a GitHub or LocalGit repository
  - View deployment logs
  - Redeploy previous deployments
```